ui/html/choose-fiat.html

```diff
@@ -22,7 +22,7 @@ <h2 class="js-i18n primary-side-title xs-margin-bottom">How much?</h2>
         <button id="choose-fiat-promo-button" class="bordered-progress-button button d2">
           Have a promo code?
         </button>
-        <p id="choose-fiat-coupon-added" class="d2" style="display:none">✔ Coupon added (5% discount)</p>
+        <p id="choose-fiat-code-added" class="d2" style="display:none">✔ Promo code added (5% discount)</p>
```

Comment:
are these 5% discount messages a placeholder? they seem to be actual hard coded discount rates
there are more examples of this in insert-bills.html and start.html

Reply:
Yup, they are placeholders. Useful for, when developing UI on the machine, having content on the static page